two_factor/views/core.py

```diff
@@ -218,6 +266,44 @@ def get_context_data(self, form, **kwargs):
             context['cancel_url'] = resolve_url(settings.LOGOUT_URL)
         return context
 
+    @cached_property
+    def remember_agent(self):
+        """
+        Returns True if a user, browser and device is remembered using the remember cookie.
+        """
+        if not getattr(settings, 'TWO_FACTOR_REMEMBER_COOKIE_AGE', None):
+            return False
+
+        user = self.get_user()
+        if user:
```

Comment:
This line has no coverage. However it's not due to a lack of tests, but rather it's only ever used in conjunction with calls to `default_device` which already checks against user being `None`.

I'm fairly sure this line can be removed and the rest of the function will work fine.